### ISSUE TITLE
Error - Too Many Requests

### ISSUE THREAD

**Issue by Under-Siege** [Sheherezadhe/awair-uploader#48]
Error while sending data to PW servers of XXXXX TOO MANY REQUESTS - Mail

Utilizing V2, getting this error very frequently now. I've tried reload, force reload, logout and log back in, still occurs very frequently.

If it matters, I'm running 40 Awair accounts with a combined 48 sensors. Unclear if this issue is actually impacting my data streams to PlanetWatch, won't know until I see my rewards tomorrow I guess.

![image](https://user-images.githubusercontent.com/102933577/161459890-96b978a2-e02a-417b-a351-34c99c751363.png)

**Comment by Under-Siege** [comment]
And again

![image](https://user-images.githubusercontent.com/102933577/161460108-655d7518-156a-40ef-bd53-89445ca0ef71.png)

**Comment by KryptoMyner** [comment]
Same here.

**Comment by gmrk33** [comment]
Same here, but I run 2 instances of the app with 2 different awair accounts, 5 sensors each. They are both running on my server.

**Comment by stefor123** [comment]
I'm in the same boat. Can I go back to V1, as it seemed to work for me?  I have 1 account with 8 Awairs.

Thanks for what you've done

**Comment by KryptoMyner** [comment]
> I'm in the same boat. Can I go back to V1, as it seemed to work for me? I have 1 account with 8 Awairs.
> 
> Thanks for what you've done

I don't think it matters which version, version 2 just tells you the errors. My version 1 was working fine, but nothing was being written to the blockchain. Then I upgraded and saw the errors in version 2, and that explains what was going on with version 1.

**Comment by FDMcreative** [comment]
Same here:
Error while sending data to PW servers ... TOO MANY REQUESTS - Data Ingestion

**Comment by Under-Siege** [comment]
Issue does seem to be impacting reward stream count with PlanetWatch. This was my 2nd day using the uploader so would've been first full day day for the reward period just paid out. Was running Version 1 the first half of the day, had maybe an hour gap where I switched from V1 to V2 and explorer only shows a gap during that time but stream count for rewards was roughly 50% across the board.

![image](https://user-images.githubusercontent.com/102933577/161543778-92c9e9de-fb08-4e26-9b53-3fa7291da138.png)

![image](https://user-images.githubusercontent.com/102933577/161543971-6df3660d-d6d5-4f2e-81c3-0afe0ffe5f90.png)

![image](https://user-images.githubusercontent.com/102933577/161544080-84d4b695-eafa-4790-8d23-0d16253ebcb0.png)

**Comment by EpHesiusLQT** [comment]
> I'm in the same boat. Can I go back to V1, as it seemed to work for me? I have 1 account with 8 Awairs.
> 
> Thanks for 

> I'm in the same boat. Can I go back to V1, as it seemed to work for me? I have 1 account with 8 Awairs.
> 
> Thanks for what you've done

![V1](https://user-images.githubusercontent.com/102968276/161544752-3b9bf936-f656-4122-a217-e0e906dfde21.PNG)
Here you have v.1 version friend =)

**Comment by TimOMalley** [comment]
I was getting the error with V2 so I went back to V1 and it's working again.

**Comment by Under-Siege** [comment]
> I was getting the error with V2 so I went back to V1 and it's working again.

The main difference between V1 and V2 was adding error details, no? lol

![image](https://user-images.githubusercontent.com/102933577/161868516-bc92c7b4-a8fd-45ba-9846-c1e193c91bae.png)

**Comment by Under-Siege** [comment]
Still receiving the same error relatively frequently. I am shutting down and relaunching the application roughly every 12 hours to avoid the token refresh issue but despite having V2 running for a full 24hours last reward payout collected < 500 streams across the board on all of my devices.

![image](https://user-images.githubusercontent.com/102933577/161868881-f9e33332-c48f-4390-bfde-bd1acd289764.png)

![image](https://user-images.githubusercontent.com/102933577/161868936-6d9317d2-b47a-4b95-b5da-4d487533d3bf.png)

![image](https://user-images.githubusercontent.com/102933577/161868977-066b4335-a815-4956-8be5-5d57893464cb.png)

![image](https://user-images.githubusercontent.com/102933577/161868998-b9132526-c30b-40be-9ac0-da8ca33f067f.png)
